haas/client/user.py

```diff
@@ -50,3 +50,22 @@ def remove(self, user, project):
         return self.check_response(
                 self.httpClient.request("POST", url, data=payload)
                 )
+
+    def set_admin(self, username, privilege):
```

Comment:
@zenhack In passing a boolean rather than a string to the client library, the check `if privilege not in('admin', 'regular'): ...` also must change.  Would it be appropriate to move the check for correct spelling to the CLI before I call `set_admin()`?